### ISSUE TITLE
data-* would be better if namespaced

### ISSUE THREAD

**Issue by CodaCodr** [ducksboard/gridster.js#11]
I'd suggest that data-col et al should be namespaced.  With such generic-sounding attrs, the likelihood of collisions with other libs and/or on-page code is quite high.

Ex: &lt;li data-grdstr-col="1" ...&gt;  &lt;!-- js := grdstrCol --&gt;

$0.02

**Comment by vieron** [comment]
I agree the idea of namespacing data-attributes, but I think it's better to keep them as are now by default and allow custom namespacing via configuration options. What do you think?

Thanks for the suggestion!

**Comment by wulczer** [comment]
Yeah, to keep backwards compatibility, I'd allow namespacing via a config option and use data-row by default (just an opinion :) )

**Comment by ScottIngram** [comment]
another $0.02... having separate data-\* attributes for each Gridster param requires a lot of typing and fills up a lot real-estate in my HTML.  I'd rather see:
data-gridster="r:1, c:1, w:2, h:3" or 
data-gridster="row:1, col:1, width:2, height:3" or
data-gridster="row:1, col:1, sizex:2, sizey:3" etc
